types/elliptic/index.d.ts

```diff
@@ -176,7 +176,7 @@ export class ec {
 
     keyPair(options: ec.KeyPairOptions): ec.KeyPair;
     keyFromPrivate(priv: Buffer | string | ec.KeyPair, enc?: string): ec.KeyPair;
-    keyFromPublic(pub: Buffer | string | ec.KeyPair, enc?: string): ec.KeyPair;
+    keyFromPublic(pub: Buffer | {x: string, y: string} | ec.KeyPair, enc?: string): ec.KeyPair;
```

Comment:
string should still be available as y type for the public key

Reply:
Fixed, thanks for catching that!